modules/updating-virtual-hardware-on-vsphere.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// updating/updating-hardware-on-nodes-running-in-vsphere.adoc
+
+[id="updating-virtual-hardware-on-vsphere_{context}"]
```

Comment:
https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere-lifecycle-manager.doc/GUID-BE02FDB4-8C09-4EEF-93F0-E0519D19FE70.html

you can schedule the virtual machine to be upgraded as well. When they reboot the virtual machine will be upgraded to the version configured.